examples/reference/custom_components/JSComponent.md

```diff
@@ -73,13 +73,16 @@ The following signatures are valid when listening to change events:
 
 - `.on('<parameter>', callback)`: Allows registering an event handler for a single parameter.
 - `.on(['<parameter>', ...], callback)`: Allows adding an event handler for multiple parameters at once.
-- `.on('change:<parameter>', callback)`: The `change:` prefix allows disambiguating change events from lifecycle hooks should a parameter name and lifecycle hook overlap.
+
+The `change:` prefix allows disambiguating change events from lifecycle hooks should a parameter name and lifecycle hook overlap.
 
 #### Lifecycle Hooks
 
-- `after_render`: Called once after the component has been fully rendered.
-- `after_resize`: Called after the component has been resized.
-- `remove`: Called when the component view is being removed from the DOM.
+- .on(`after_render`, callback): Called once after the component has been fully rendered.
```

Comment:
Are you sure you want to call it `after_render` and not `after_layout`? I mixed them up because its `after_layout` when working directly with bokeh models. I.e. it might be mental overhead at least for some users with different names.